code-studio/src/js/code-studio.js

```diff
@@ -9,6 +9,9 @@
 'use strict';
 /* global Sounds */
 
+// Shim window.console to be safe in IE
+require('./consoleShim')(window);
```

Comment:
Some ideas:

A) Let `require` have a side effect.  This seems to be what most such shims do, but I'm not a fan.
```javascript
require('./consoleShim'); // MAGIC
```

B) Export a function (what I'm doing right now)
```javascript
// Can invoke directly, which is a awkward but makes it obvious
// that 'work' is being done
require('./consoleShim')(window)

// Or do a two-liner, which is a classier but some people might 
// think it's over-verbose here.
var consoleShim = require('./consoleShim');
consoleShim(window);
```

C) Export a cloned/wrapped console object.  I like how this makes the operation explicit, but now  consoleShim depends on global window as in A, and I wonder what magic we have to do to avoid breaking anything here (doesn't window.console has some privileged behavior on some browsers?)
```javascript
window.console = require('./consoleShim');
```

What do you think?